### PR TITLE
flexible journald source

### PR DESCRIPTION
**Pull request by rockb1017**

### CODE REVIEW

File: charts/sck-otel/values.yaml [hunk 1]
```diff
@@ -176,12 +176,15 @@ journaldLogs:
...
-    - ssh
-    - kubelet
-    - docker
-    - containerd
+  - name: kubelet
+    priority: info
+  - name: docker
+    priority: info
+  - name: containerd
+    priority: info
```
Comment (luckyj5) [inline charts/sck-otel/values.yaml:187]: To ingest all the journald service logs, do we leave `units` blank like below? 

> `units: ` 

What are the default values for units?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,7 +8,8 @@
 ### 💡 Enhancements 💡
 - This chart now can send logs (#62) and metrics (#64) to both Splunk platform and Observerability suite
 - Support trace data pipeline (#64)
-
+- Flexiable Journald log source (#76)
```
Comment (luckyj5) [inline CHANGELOG.md]: Typo fix- `Flexible`

File: charts/sck-otel/values.yaml (hunk 1 above)
Comment (rockb1017) [reply charts/sck-otel/values.yaml:187]: yes. i left a comment to leave it blank to collect all.

File: CHANGELOG.md (hunk 2 above)
Comment (rockb1017) [reply CHANGELOG.md]: Thanks!

File: charts/sck-otel/values.yaml (hunk 1 above)
Comment (luckyj5) [reply charts/sck-otel/values.yaml:187]: Are we adding only "kubelet", "docker" and "containerd" units by default? or all?

Comment (rockb1017) [reply charts/sck-otel/values.yaml:187]: yea, just following what we had in SCK

**Review by luckyj5 — APPROVED** [review]
Thanks!

**Pull request merged**